### PR TITLE
Bai1 intent filter duy lh 2016/11/10

### PR DESCRIPTION
**Pull request by duyle112**

### CODE REVIEW

File: app/src/main/java/vn/asiantech/training/DemoIntentFilter.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    private Button mBtnsendEmail;
+    private Button mBtncamera;
+    private ImageView mImgdisplay;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_demo_intent_filter);
+        init();
+
```
Comment (at-android) [inline app/src/main/java/vn/asiantech/training/DemoIntentFilter.java]: Remove this line

File: app/src/main/res/layout/activity_demo_intent_filter.xml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    <Button
+        android:id="@+id/Btncall"
+        style="@style/activity_demo_intent"
+        android:layout_width="@dimen/activity_demo_intent_btn_width"
+        android:layout_height="wrap_content"
+        android:text="@string/btn_call" />
+
+    <Button
+        android:id="@+id/Btnmessage"
```
Comment (at-android) [inline app/src/main/res/layout/activity_demo_intent_filter.xml]: change Btnmessage to btnMessage

File: app/src/main/res/layout/activity_demo_intent_filter.xml [hunk 3]
```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="vn.asiantech.training.DemoIntentFilter">
+
+    <Button
+        android:id="@+id/Btncall"
```
Comment (at-android) [inline app/src/main/res/layout/activity_demo_intent_filter.xml]: Convention don't correct 
change Btncall to btnCall

File: app/src/main/res/layout/activity_demo_intent_filter.xml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        style="@style/activity_demo_intent"
+        android:layout_width="@dimen/activity_demo_intent_btn_width"
+        android:layout_height="wrap_content"
+        android:layout_toEndOf="@+id/Btncall"
+        android:layout_toRightOf="@+id/Btncall"
+        android:text="@string/btn_message"
+        tools:ignore="RtlCompat" />
+
+    <Button
+        android:id="@+id/BtnsendEmail"
```
Comment (at-android) [inline app/src/main/res/layout/activity_demo_intent_filter.xml]: Convention don't correct

File: app/src/main/res/layout/activity_demo_intent_filter.xml [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        style="@style/activity_demo_intent"
+        android:layout_width="@dimen/activity_demo_intent_btn_width"
+        android:layout_height="wrap_content"
+        android:layout_toEndOf="@+id/Btnmessage"
+        android:layout_toRightOf="@+id/Btnmessage"
+        android:text="@string/btn_sendMail" />
+
+    <Button
+        android:id="@+id/BtnlaunchWeb"
```
Comment (at-android) [inline app/src/main/res/layout/activity_demo_intent_filter.xml]: Convention don't correct

File: app/src/main/res/layout/activity_demo_intent_filter.xml [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+    <Button
+        android:id="@+id/BtnopenGGstore"
+        style="@style/activity_demo_intent"
+        android:layout_width="@dimen/activity_demo_intent_btn_width"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/Btncall"
+        android:text="@string/btn_openGGstore" />
+
+    <Button
+        android:id="@+id/BtnopenGGmap"
```
Comment (at-android) [inline app/src/main/res/layout/activity_demo_intent_filter.xml]: Convention don't correct

File: app/src/main/res/layout/activity_demo_intent_filter.xml [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+        style="@style/activity_demo_intent"
+        android:layout_width="@dimen/activity_demo_intent_btn_width"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/Btnmessage"
+        android:layout_toEndOf="@+id/BtnopenGGstore"
+        android:layout_toRightOf="@+id/BtnopenGGstore"
+        android:text="@string/btn_openGGmap" />
+
+    <Button
+        android:id="@+id/BtnopenGallery"
```
Comment (at-android) [inline app/src/main/res/layout/activity_demo_intent_filter.xml]: Convention don't correct

File: app/src/main/res/layout/activity_demo_intent_filter.xml [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+        style="@style/activity_demo_intent"
+        android:layout_width="@dimen/activity_demo_intent_btn_width"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/BtnsendEmail"
+        android:layout_toEndOf="@+id/BtnopenGGmap"
+        android:layout_toRightOf="@+id/BtnopenGGmap"
+        android:text="@string/btn_openGallery" />
+
+    <Button
+        android:id="@+id/Btncamera"
```
Comment (at-android) [inline app/src/main/res/layout/activity_demo_intent_filter.xml]: Convention don't correct

File: app/src/main/res/values/colors.xml [hunk 9]
```diff
@@ -3,4 +3,7 @@
     <color name="colorPrimary">#3F51B5</color>
     <color name="colorPrimaryDark">#303F9F</color>
     <color name="colorAccent">#FF4081</color>
+    <color name="Green">#00CC00</color>
```
Comment (ghost) [inline app/src/main/res/values/colors.xml]: color em dat ten nen như : colorGreen

File: app/src/main/res/values/colors.xml [hunk 10]
```diff
@@ -3,4 +3,7 @@
...
+    <color name="Black">#000000</color>
```
Comment (ghost) [inline app/src/main/res/values/colors.xml]: nen dung mau macj dinh @android:color/black, chinh lai nhung cho nao dung mau nay nhe em

**Pull request closed without merging**